python/test/gluon/test_frontend.py

```diff
@@ -446,6 +447,48 @@ def test_tcgen05_mma(fresh_knobs):
 """)
 
 
+@gluon.jit
+def warpgroup_mma_kernel(nvmma_layout: ttgl.constexpr, acc_layout: ttgl.constexpr):
+    a = ttgl.allocate_shared_memory(ttgl.float16, [128, 128], nvmma_layout)
+    b = ttgl.allocate_shared_memory(ttgl.float16, [128, 128], nvmma_layout)
+    acc = ttgl.zeros([128, 128], dtype=ttgl.float16, layout=acc_layout)
```

Comment:
NIT: The function call is making the IR a lot worse.
```suggestion
    acc = ttgl.full([128, 128], 0, dtype=ttgl.float16, layout=acc_layout)
```